docs/operating-scylla/nodetool.rst

```diff
@@ -14,9 +14,9 @@ Nodetool
    nodetool-commands/cleanup
    nodetool-commands/clearsnapshot
    nodetool-commands/cluster/index
+   nodetool-commands/compact
    nodetool-commands/compactionhistory
    nodetool-commands/compactionstats
-   nodetool-commands/compact
```

Comment:
Reorderings and in general refactorings like this should be done as a separate commit, not attached to another change.